cookbooks/simple_ice_box.prm

```diff
@@ -0,0 +1,174 @@
+
+    # Choose to have the viscosity (pre-yield) follow a dislocation
+    # diffusion or composite flow law.  Here, dislocation is selected
+    # so no need to specify diffusion creep parameters below, which are
+    # only used if "diffusion" or "composite" option is selected.
+    set Viscous flow law = dislocation
+
+    # Dislocation creep parameters for
+    # Note that the viscous pre-factors below are scaled to plane strain from unixial strain experiments.
+    set Prefactors for dislocation creep          =   1.667e-7  
+    set Stress exponents for dislocation creep    =      4.8
+    set Activation energies for dislocation creep =    60e+3
+    set Activation volumes for dislocation creep  =        0
+
+    # Plasticity parameters
+    set Angles of internal friction =   30.
+    set Cohesions                   =  1e7     
+
+  end
+end
+
+
+subsection Geometry model
+  set Model name = box
+
+  subsection Box
+    set X repetitions = 100
+    set Y repetitions = 25
+    set X extent = 100e3
+    set Y extent = 25e3
+  end
+end
+
+subsection Free surface
+  set Free surface boundary indicators        = top
+  set Surface velocity projection = vertical
+end 
+
+# Velocity on boundaries characterized by functions
+# The outward velocity (x-direction) on the left and right walls is 1.25 cm/year
+# The vertical velocity at the base is 0.125 cm/year (balances outflow on sides)
+# Velocity components parallel to the base (x-velocity) and side walls (y-velocity)
+# are unconstrained (i.e. 'free').
+subsection Boundary velocity model 
+  set Prescribed velocity boundary indicators = left x: function, right x:function, bottom:function
+
+  subsection Function
+    set Variable names      = x,y
+    set Function constants  = cm=0.01, year=1
+    set Function expression = if (x<50e3 ,     0*cm/year,    0*cm/year) ; 0          
+  end
+end
+
+subsection Boundary temperature model
+  set Fixed temperature boundary indicators   = top
+  set List of model names = initial temperature
+end
+
+#This module introduces the heat flux boundary condition
+subsection Boundary heat flux model
+  set Fixed heat flux boundary indicators = bottom
+  set Model name = function
+
+  subsection Function
+    set Function expression = -0.04
+  end
+end
+
+#A variable surface tempreature is prescribed
+subsection Boundary temperature model
+  set Fixed temperature boundary indicators = top
+  set List of model names = function
+  subsection Function
+  set Variable names = x,y
+#    set Top temperature =  90
+   set Function expression = 60+30*sin(asin((x/100000)))^.25
+  end
+end
+
+
+subsection Initial temperature model
+  set Model name = function
+  subsection Function
+    set Variable names = x,y
+    set Function expression = 60+30*sin(asin((x/100000)))^.25 + (25.e3-y)*0.00732;
+    end 
+end
+
+subsection Solver parameters
+  subsection Stokes solver parameters
+    set Linear solver tolerance = 1e-8
+    end
+end
+
+
+# Gravity model
```

Comment:
Perhaps add a comment here indicating what this gravity value is based on (e.g., which planet)?